web/src/settings_invites.ts

```diff
@@ -148,7 +148,7 @@ function do_revoke_invite({
 }): void {
     const modal_invite_id = $(".dialog_submit_button").attr("data-invite-id");
     const modal_is_multiuse = $(".dialog_submit_button").attr("data-is-multiuse");
-    const $revoke_button = $row.find("button.revoke");
+    const $revoke_button = $row.find("button.revoke-invite");
```

Comment:
This button change commit could be usefully pulled out as its own preparatory PR.